pkg/webhook/uniteddeployment/validating/uniteddeployment_validation.go

```diff
@@ -340,6 +342,19 @@ func validateSubsetTemplate(template *appsv1alpha1.SubsetTemplate, selector labe
 			return allErrs
 		}
 		allErrs = append(allErrs, appsvalidation.ValidatePodTemplateSpecForReplicaSet(coreTemplate, nil, selector, 0, fldPath.Child("deploymentTemplate", "spec", "template"), webhookutil.DefaultPodValidationOptions)...)
+	} else if template.CloneSetTemplate != nil {
+		labels := labels.Set(template.CloneSetTemplate.Labels)
+		if !selector.Matches(labels) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("cloneSetTemplate", "metadata", "labels"), template.CloneSetTemplate.Labels, "`selector` does not match template `labels`"))
+		}
+		allErrs = append(allErrs, validateCloneSet(template.CloneSetTemplate, fldPath.Child("cloneSetTemplate"))...)
+		template := template.CloneSetTemplate.Spec.Template
+		coreTemplate, err := convertor.ConvertPodTemplateSpec(&template)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath.Root(), template, fmt.Sprintf("Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec failed: %v", err)))
+			return allErrs
+		}
+		allErrs = append(allErrs, appsvalidation.ValidatePodTemplateSpecForReplicaSet(coreTemplate, selector, 0, fldPath.Child("cloneSetTemplate", "spec", "template"), webhookutil.DefaultPodValidationOptions)...)
```

Comment:
pkg/webhook/uniteddeployment/validating/uniteddeployment_validation.go:357:163: not enough arguments in call to appsvalidation.ValidatePodTemplateSpecForReplicaSet